frontend/packages/dev-console/src/components/import/DeployImageForm.tsx

```diff
@@ -36,6 +37,7 @@ const DeployImageForm: React.FC<FormikProps<FormikValues> & DeployImageFormProps
       <AppSection project={values.project} />
       <ImageSearchSection />
       <ServerlessSection />
+      <RouteCheckbox />
```

Comment:
Everything else above is a section and this is a standalone form item.
The sections above it all include separators (i'd prefer that they didn't and have the parent form put separators between items).
Where's the design for this? 
This won't have a separator between it and the next form item.

Reply:
The design is linked to the jira linked above. But here it is directly.
https://docs.google.com/document/d/1kqKQxw55GBt6V4jBnF8uh22A_BimcoAvr4wOMSDX-e4/edit

Reply:
Looks like there are other discrepancies wrt dividers and order with the designs that'll need to be addressed separately.